kani-compiler/src/codegen_cprover_gotoc/codegen/typ.rs

```diff
@@ -487,9 +488,20 @@ impl<'tcx> GotocCtx<'tcx> {
         // This is not a restriction because C can only access non-generic types anyway.
         // TODO: Skipping name mangling is likely insufficient if a dependent crate has two versions of
         // linked C libraries
-        // https://github.com/model-checking/kani/issues/450
+        // https://github.com/model-checking/kani/issues/450.
+        // Note that we only mangle #[repr(C)] types if the unstable c-ffi feature is enabled; otherwise,
```

Comment:
Should this be "we **don't** mangle #[repr(C)] types if the unstable c-ffi feature is enabled"?